### PR TITLE
run btn bug corrected

### PR DESCRIPTION
**Pull request by 0xkafkaa**
#13 Corrected the bug for run btn issue

Regard,
0xkafkaa.

### CODE REVIEW

File: anadi/layouts/horizontal_layout.tcss [hunk 1]
```diff
@@ -43,6 +43,7 @@ CommandContainer {
     dock: right;
     border: none;
     height: 1;
+	margin-top: 1;
```
Comment (Cereal84) [inline anadi/layouts/horizontal_layout.tcss:46]: This param has not correct identation

Comment (Cereal84) [inline anadi/layouts/horizontal_layout.tcss]: This param has not correct identation.

**Review by Cereal84 — CHANGES_REQUESTED** [review]

File: anadi/layouts/horizontal_layout.tcss (hunk 1 above)
Comment (0xkafkaa) [reply anadi/layouts/horizontal_layout.tcss]: Its corrected now.

**Review by Cereal84 — APPROVED** [review]

**Comment by Cereal84** [comment]
Thank you @0xkafkaa

**Pull request merged**